dynawo/sources/Models/CPP/ModelVariationArea/DYNModelVariationArea.cpp

```diff
@@ -145,7 +147,7 @@ ModelVariationArea::evalF(const double & t) {
 
 void
 ModelVariationArea::evalG(const double & t) {
-  gLocal_[0] = (t - startTime_) >= 0 ? ROOT_UP : ROOT_DOWN;
+  gLocal_[0] = ((t - startTime_) >= 0 && (t - stopTime_) <= 0) ? ROOT_UP : ROOT_DOWN;
```

Comment:
To be exclusive with the other mode, I would have had a < or a > for the next one but here at t = stopTime, both are activated and finished is taken finally only due to the c++ code order in evalMode